stable/kuberhealthy/README.md

```diff
@@ -68,6 +79,41 @@ securityContext: # default container security context
   allowPrivilegeEscalation: false
 ```
 
+The following table lists the configurable parameters of the kuberhealthy chart and their default values.
```

Comment:
Added a table here for the configurable parameters, not sure if this is preferred over what we already have!

Reply:
Love it!